modules/openapi-generator/src/main/resources/csharp-netcore/modelAnyOf.mustache

```diff
@@ -208,7 +208,10 @@
         /// <returns>The object converted from the JSON string</returns>
         public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
         {
-            return {{classname}}.FromJson(JObject.Load(reader).ToString(Formatting.None));
+            if(reader.TokenType != JsonToken.Null){
```

Comment:
@Ghufz thanks for the PR. Can we make it conform to C# code style? e.g
```csharp
if (reader.TokenType != JsonToken.Null)
{
    return {{classname}}.FromJson(JObject.Load(reader).ToString(Formatting.None));
}
return null;
```

Reply:
done